packages/tupaia-web/src/Routes.tsx

```diff
@@ -92,7 +134,7 @@ export const Routes = () => {
           <Route path="/:projectCode/:entityCode" element={<ProjectPageDashboardRedirect />} />
 
           {/* The Dashboard has to be rendered below the Map, otherwise the map will re-mount on route changes */}
-          <Route path={ROUTE_STRUCTURE} element={<Dashboard />} />
+          <Route path={ROUTE_STRUCTURE} element={<DashboardNameToCodeRedirect />} />
```

Comment:
This is pretty minor but I was initially confused by the naming of DashboardNameToCodeRedirect as I didn't realise that it included the Dashboard component. Maybe it should be just called DashboardRoute or DashboardCodeRoute or something?